### PR TITLE
Kneebreaker Hammer and ported biosuits

### PR DESCRIPTION
**Pull request by Fire-and-Glory**
Adding the Kneebreaker Hammer to the code, a blunt weapon which does the same amount of damage as a metal baseball bat and can be used to pry doors while fitting in a satchel and on the belt, quite comparable to an e-sword, due to eventually become a traitor item when the coders are done with Stuff.

Porting our oldcode biosuits as well, when I made some checks, those who chose to comment unanimously agreed the old suits were better.

### CODE REVIEW

**Comment by Fire-and-Glory** [comment]
I haven't been able to get the <span> to work and I am tired, given the superfluous nature of giving it suicide text, I am content to simply axe it.

File: code/game/objects/items/weapons/melee/misc.dm [hunk 1]
```diff
@@ -53,3 +53,26 @@
...
+	color
+	name = "kneebreaker hammer"
+	desc = "An intimidating hammer made of plasteel, this would hurt."
+	icon = 'icons/obj/kneehammer.dmi'
+	icon_state = "kneehammer"
+	item_state = "kneehammer"
+	contained_sprite = 1
+	slot_flags = SLOT_BELT
+	force = 22
+	throwforce = 70
```
Comment (Alberyk) [inline code/game/objects/items/weapons/melee/misc.dm]: I am nearly sure that 70 of force means 70 of brute when throw

File: code/game/objects/items/weapons/melee/misc.dm [hunk 2]
```diff
@@ -53,3 +53,26 @@
 /obj/item/weapon/melee/chainsword/suicide_act(mob/user)
 	viewers(user) << "\red <b>[user] is slicing \himself apart with the [src.name]! It looks like \he's trying to commit suicide.</b>"
 	return (BRUTELOSS|OXYLOSS)
+
+
+//This is essientially a crowbar and a baseball bat in one.
+/obj/item/weapon/melee/misc/hammer
+	color
```
Comment (Alberyk) [inline code/game/objects/items/weapons/melee/misc.dm]: What is this for?

File: code/game/machinery/doors/airlock.dm [hunk 3]
```diff
@@ -896,10 +896,21 @@ About the new airlock wires panel:
 					spawn(0)	close(1)
 				else
 					user << "<span class='warning'>You need to be wielding \the [C] to do that.</span>"
+
+	else if(istype(C, /obj/item/weapon/melee/misc/hammer) && !arePowerSystemsOn())
```
Comment (Alberyk) [inline code/game/machinery/doors/airlock.dm]: Maybe you can make the hammer a child of the crowbar and avoid adding those?

File: code/game/objects/items/weapons/melee/misc.dm (hunk 2 above)
Comment (Fire-and-Glory) [inline code/game/objects/items/weapons/melee/misc.dm]: No idea, I was wondering what was causing that to show up in the spawnable items list.

File: code/game/objects/items/weapons/melee/misc.dm (hunk 1 above)
Comment (Fire-and-Glory) [inline code/game/objects/items/weapons/melee/misc.dm]: You would think, but in my own testing 22 throw force does scratch damage in the single-digit range, while 70 changes between 13 to 17 brute.

File: code/game/machinery/doors/airlock.dm (hunk 3 above)
Comment (Fire-and-Glory) [inline code/game/machinery/doors/airlock.dm]: I gave it a bit of thought and talk on Discord and decided that giving the hammer the same abilities of the crowbar (like the incredibly irritating occurrence in combat where the crowbar is constantly prying floor tiles when you miss) would be a really bad idea.

**Pull request merged**